### ISSUE TITLE
How to use events

### ISSUE THREAD

**Issue by cfjedimaster** [pixelastic/algolia-indexing#4]
The docs mention certain events are emitted, but there is no documentation on how to use them. For example, I tried `indexer.on(....)` but that didn't work. Neither did `addEventListener`.

**Comment by pixelastic** [comment]
I just released `v1.0.1` which exposes the `.pulse` key on the module, and updated the documentation accordingly.

```javascript
const indexing = require('algolia-indexing');

indexing.pulse.on('copyIndex.start', ({ source, destination }) => {
  console.info(`Start moving ${source} to ${destination}`);
});
indexing.pulse.on('copyIndex.end', ({ source, destination }) => {
  console.info(`Finished moving ${source} to ${destination}`);
});
```

**Comment by cfjedimaster** [comment]
I was definitely getting an error. Unfortunately I can't seem to recreate it now.

**Comment by pixelastic** [comment]
I think I fixed it in the v1.0.1 as well. It was erroring when trying to get records from a not-yet-existing index. Should work now :)
